faceUnlocker.py

```diff
@@ -0,0 +1,106 @@
+import cv2
+import numpy as np
+import face_recognition as fc
+import os
+
+
+
+def unlock():
+    image = os.path.join('asset','img','login','')
+
+    files = []
+    for root,file,dirs in os.walk(image):
+        for _dir in dirs:
+            files.append(_dir)
+
+    if len(files) > 1:
```

Comment:
Also add exception handling here, if no images are found then instead of displaying error it shows a message to add an image. Also if the `login` folder does not exist create it.